docs/installation/upgrading.md

```diff
@@ -28,44 +28,44 @@ Download and extract the latest version:
 # wget https://github.com/netbox-community/netbox/archive/vX.Y.Z.tar.gz
 # tar -xzf vX.Y.Z.tar.gz -C /opt
 # cd /opt/
-# ln -sfn netbox-X.Y.Z/ netbox
+# ln -sfn /opt/netbox-X.Y.Z/ /opt/netbox
 ```
 
 Copy the 'configuration.py' you created when first installing to the new version:
 
 ```no-highlight
-# cp netbox-X.Y.Z/netbox/netbox/configuration.py netbox/netbox/netbox/configuration.py
+# cp /opt/netbox-X.Y.Z/netbox/netbox/configuration.py /opt/netbox/netbox/netbox/configuration.py
 ```
 
 Copy your local requirements file if used:
 
 ```no-highlight
-# cp netbox-X.Y.Z/local_requirements.txt netbox/local_requirements.txt
+# cp /opt/netbox-X.Y.Z/local_requirements.txt /opt/netbox/local_requirements.txt
 ```
 
 Also copy the LDAP configuration if using LDAP:
 
 ```no-highlight
-# cp netbox-X.Y.Z/netbox/netbox/ldap_config.py netbox/netbox/netbox/ldap_config.py
+# cp /opt/netbox-X.Y.Z/netbox/netbox/ldap_config.py /opt/netbox/netbox/netbox/ldap_config.py
 ```
 
 Be sure to replicate your uploaded media as well. (The exact action necessary will depend on where you choose to store your media, but in general moving or copying the media directory will suffice.)
 
 ```no-highlight
-# cp -pr netbox-X.Y.Z/netbox/media/ netbox/netbox/
+# cp -pr /opt/netbox-X.Y.Z/netbox/media/ netbox/netbox/
```

Comment:
This destination path should be absolute as well, to keep things consistent.

Reply:
You're absolutely correct